exercises/tournament/canonical-data.json

```diff
@@ -191,6 +191,25 @@
         "Blithering Badgers             |  3 |  0 |  1 |  2 |  1",
         "Devastating Donkeys            |  3 |  0 |  1 |  2 |  1"
       ]
+    },
+    {
+      "uuid": "f9e20931-8a65-442a-81f6-503c0205b17a",
+      "description": "ensure points sorted numerically",
+      "property": "tally",
+      "input": {
+        "rows": [
+          "A;B;win",
+          "A;B;win",
+          "A;B;win",
+          "A;B;win",
+          "A;B;loss"
```

Comment:
All the other test cases use made-up names for the teams. Maybe we should be consistent and use them here too?

Reply:
That be a nice change, yes.